core/src/main/java/nl/nn/adapterframework/webcontrol/api/ShowConfigurationStatus.java

```diff
@@ -719,15 +719,17 @@ private Map<String, Object> mapAdapter(Adapter adapter) {
 				try {
 					errorStoreMessageCount += esmb.getMessageCount();
 				} catch (ListenerException e) {
-					log.warn("Cannot determine number of messages in errorstore of ["+rcv.getName()+"]", e);
+					if(log.isInfoEnabled()) log.warn("Cannot determine number of messages in errorstore of [{}]", rcv.getName(), e);
+					else log.warn("Cannot determine number of messages in errorstore of [{}]: {}", rcv::getName, e::getMessage);
 				}
 			}
 			IMessageBrowser<?> mlmb = rcv.getMessageBrowser(ProcessState.DONE);
 			if(mlmb != null) {
 				try {
 					messageLogMessageCount += mlmb.getMessageCount();
 				} catch (ListenerException e) {
```

Comment:
```suggestion
				} catch (ListenerException e) {
						// Only log the stacktrace when loglevel == INFO. Otherwise it will pollute the log too much.
```